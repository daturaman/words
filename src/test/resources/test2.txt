**** Please don't count asterisks, they're not allowed! ******