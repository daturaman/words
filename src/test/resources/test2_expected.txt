Word count = 7
Average word length = 4.556
Number of words of length 3 is 1
Number of words of length 4 is 1
Number of words of length 5 is 1
Number of words of length 6 is 1
Number of words of length 1 is 1
Number of words of length 9 is 1
The most frequently occurring word length is 2, for word lengths of 6